dynamic/version/version.go

```diff
@@ -15,6 +15,11 @@
 package version
 
 // version is initialized by the Go linker to contain the semver of this build.
+
```

Comment:
The original comment on `version` should still be attached to the variable.
```suggestion
//
```